docs/contributing.rst

```diff
@@ -1,4 +1,197 @@
 Contributing to AllTheBacteria
 ==============================
 
-To do
+We welcome contributions to AllTheBacteria. Please read on for instructions
+on how to contribute to the project.
+
+Background information
+----------------------
+
+Where does everything live?
```

Comment:
Maybe better to say "Where do all the files live?" or "Where is the data located?"